aiohttp_retry/client.py

```diff
@@ -80,13 +80,18 @@ def __init__(
 
         self._response: Optional[ClientResponse] = None
 
-    def _is_status_code_ok(self, code: int) -> bool:
-        if code >= 500 and self._retry_options.retry_all_server_errors:
-            return False
-        return code not in self._retry_options.statuses
+    async def _skip_retry(self, response: ClientResponse) -> bool:
```

Comment:
I would suggest a name ```_is_skip_retry```
This way name state that it return a bool

Reply:
Maybe name as ```_is_new_try``` or ```_is_stop_retrying``` ?

Reply:
if we name it _is_new_try we have to invert the logic of evaluate_response_callback. Also for me this is the better outcome since the callback can then return True if you want to retry. Now we return False if we want to retry which feels a bit odd. Also I would rename evaluate_response_callback to something like evaluate_is_retry_callback.
Otherwise I'm ok with _is_skip_retry